packages/puppeteer-core/src/bidi/Page.ts

```diff
@@ -774,6 +775,136 @@ export class BidiPage extends Page {
     });
   }
 
+  override async cookies(...urls: string[]): Promise<Cookie[]> {
+    const normalizedUrls = (urls.length ? urls : [this.url()]).map(url => {
+      return new URL(url);
+    });
+
+    const bidiCookies = await this.#connection.send('storage.getCookies', {
+      partition: {
+        type: 'context',
+        context: this.mainFrame()._id,
+      },
+    });
+    return bidiCookies.result.cookies
+      .map(c => {
+        return this.#bidiToPuppeteerCookie(c);
+      })
+      .filter(c => {
+        return normalizedUrls.some(u => {
+          return BidiPage.#testUrlMatchCookie(c, u);
```

Comment:
let's use full name `cookie` and `url` for readbility

Reply:
done